src/pages/ArchivePage.spec.js

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import ArchivePage from './ArchivePage';
+
+describe('ArchivePage', () => {
+  it('renders a page with the heading "archive" and two archived items', () => {
+    const archivedWishes = [
+      {
+        id: '1',
+        destination: 'Maldives',
+        notes: 'I want to go there for my next diving holiday',
+        isArchived: true,
+      },
+      {
+        id: '2',
+        destination: 'Galapagos Island',
+        notes: 'My dream destination but first I have to win the lottery.',
+        isArchived: true,
+      },
+    ];
+
+    render(
+      <MemoryRouter>
```

Comment:
Do you need the MemoryRouter for testing the single component ArchivePage and to render the content?

Reply:
Yes, as I have the useNavigate hook inside the ArchivePage. Without the MemoryRouter I will get an error while testing